src/index.ts

```diff
@@ -0,0 +1,74 @@
+import * as t from "./interfaces";
+import Output from "./output";
+import ReadableDir from "./readable_dir";
+import TempDir from "./temp_dir";
+import { makeTempDir } from "./util";
+
+export * from "./interfaces";
+
+/**
+ * Wrap a fixture directory in a `ReadableDir` interface.
+ * @param dir
+ *
+ * @public
+ */
+export function wrapDir(dir: string): t.ReadableDir {
```

Comment:
just spit-balling, what about `fromDir` instead of `wrapDir`

Reply:
this api isn't new, I could do that and deprecate wrap versions, I do like from better.

Reply:
if you also like it better, lets transition to `fromDir`.